src/api/firebase.js

```diff
@@ -224,10 +225,35 @@ export async function updateItem(listPath, itemId, dateLastPurchased) {
 	});
 }
 
-export async function deleteItem() {
+export async function deleteItem(listPath, itemId) {
```

Comment:
@MiliMade / @amandaguan-ag why are these changes in your PR?

Reply:
We initialy opened new branch` on `lc-km-no-empty-items-or-duplicates` @MiliMade Did you merge from `gl-lc-delete-item` in commit 7a24d19?